tests/app/services/test_request_supplementary_data.py

```diff
@@ -145,9 +179,13 @@ def test_get_supplementary_data_retries_transient_error(app: Flask, mocker):
             mocker, status_codes=[500, 500, 200]
         )
 
+        mocker.patch(
+            "app.services.supplementary_data.decrypt_supplementary_data",
+            return_value=decrypted_mock_supplementary_data_payload,
+        )
         mocker.patch(
             "app.services.supplementary_data.validate_supplementary_data",
-            return_value=mock_supplementary_data_payload,
+            return_value=decrypted_mock_supplementary_data_payload,
```

Comment:
this might not work - but now that you're patching the decrypt to have the correct data, do you still need to patch the `validate_supplementary_data` method?

Reply:
Great point! Have removed that mock